src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">netstandard2.0;net45;net461</TargetFrameworks>
+    <TargetFrameworks>net45;net461;netstandard2.0</TargetFrameworks>
```

Comment:
If you mean the piece of moving netstandard2.0 to the end vs. the beginning, yes (had something to do with getting it to compile/run for webforms samples/site - there's something with the default being the first and what not).  The removing the equals condition on OS platform I can't say would have been on purpose, but either way I've just added it back and re-shuffled the order again (since we likely won't be working with WebForms directly much in the future hopefully).

Reply:
Makes sense. I think I've hit that issue a few times with the first one being the default. Didn't realize this was causing it until now! Feel free to change the order of target frameworks if that helps.

The OS platform check we need because we can't build `net45` or `net461` on Linux ([yet 🔗](https://github.com/dotnet/designs/blob/1c2baaf1773e52e5c13f94b8659de377c1f16671/accepted/targeting-packs/targeting-packs.md)).